module/Application/view/layout/layout.phtml

```diff
@@ -58,17 +56,26 @@
 
     <footer>
         <div class="grid">
-            <div class="unit one-third center-on-mobiles">
+            <div class="unit half center-on-mobiles">
```

Comment:
Why are you changing from 1:2 to 1:1 ratios for the columns?

Reply:
It's in footer, we had before two columns with width: 'one-third' + 'half', now we have both 'half'. We need wider column there, because of icons I've added in footer - please see image attached in the description of the PR.

Reply:
Previously, we had two columns, one as one-third and one as two-thirds; the next diffed section demonstrates the change from two-thirds to half!

However, if we need a wider column, that's enough justification.

Reply:
Oh, yeah, you have right, sorry. We didn't use the whole space in right column so it's not breaking anything. Now both have the same width: 50-50. Tested ;-)